platformcomponents/desktop/reaction-desktop.json

```diff
@@ -0,0 +1,10 @@
+{
+  "reactionDesktop": {
+    "comment": "Use for see through backgrounds",
```

Comment:
name of the file and enum should not include platform. 
adding reactions to the desktop folder means it will only be generated on desktop